pipeline/merger/src/main/scala/uk/ac/wellcome/platform/merger/rules/ItemsRule.scala

```diff
@@ -24,21 +24,20 @@ object ItemsRule extends FieldMergeRule with MergerLogging {
     target: UnidentifiedWork,
     sources: Seq[TransformedBaseWork]): FieldMergeResult[FieldData] = {
     val rules =
-      List(mergeCalmItems, mergeMetsItems, mergeMiroItems)
+      List(
```

Comment:
Does the comment above this rule need updating? Github won't let me comment outside of the changes...

Reply:
> Github won't let me comment outside of the changes...

You have to comment on the left side of the split.